man/usr/share/man/man1m/piadm.1m.md

```diff
@@ -69,6 +69,11 @@ piadm(1M) -- Manage SmartOS Platform Images
     itself without issue.  Occasionally, however, a PI will have Boot Image
     changes also that need to accompany it.
 
+    Boot images can be customized by providing loader.conf.local and/or
+    loader.rc.local files in POOL/boot/common/. When a new boot image is
+    installed using `piadm install` these files are symlinked into the
+    images.
+
 ## BOOTABLE POOLS
 
     A SmartOS bootable pool (POOL in the examples) contains:
```

Comment:
Toward the bottom of this list, you should add a common/ directory (and mention that -r can create it if it's not there already), as well as document that the two *.local files will be symlinked to by all boot images.  (It may involve moving your new text above, OR repeating yourself, to make the point.)